### PR TITLE
Prepare for release 13 final

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ nav:
       - Bootstrap Volto: 'getting-started/install.md'
       - Developer roadmap: 'getting-started/roadmap.md'
       - Learning resources: 'getting-started/others.md'
+      - How does Volto works under the hood: 'getting-started/howdoesitwork.md'
```
Comment (tiberiuichim) [inline docs/mkdocs.yml:26]: `How Volto works under the hood`

File: docs/source/getting-started/howdoesitwork.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# How does Volto works under the hood
+
+You can watch the talk during the World Plone Day 2021:
```
Comment (nzambello) [inline docs/source/getting-started/howdoesitwork.md:3]: Shouldn't we add a plain link to the YT video to work w/o JS or to watch it externally?

Comment (nzambello) [reply docs/source/getting-started/howdoesitwork.md:3]: Maybe the embed is doing that already 🤔

Comment (sneridagh) [reply docs/source/getting-started/howdoesitwork.md:3]: I wanted to explain a bit more with narrative, but not in the mood today and left it for the near future :P 

I can turn it into a plain link.

**Review by nzambello — APPROVED** [review]
Yes, 13 is here!

File: docs/source/getting-started/howdoesitwork.md (hunk 2 above)
Comment (nzambello) [reply docs/source/getting-started/howdoesitwork.md:3]: I like the embed, I was wondering on an additional link for users with js disabled or for a11y.
Like a link on `watch the talk`.

But yes, not blocking for sure

Comment (sneridagh) [reply docs/source/getting-started/howdoesitwork.md:3]: Although it renders fine...

image

Comment (nzambello) [reply docs/source/getting-started/howdoesitwork.md:3]: All right, there's a link 👍

Comment (sneridagh) [reply docs/source/getting-started/howdoesitwork.md:3]: yep!

**Pull request merged**